pkg/util/template/func_test.go

```diff
@@ -185,7 +185,7 @@ func TestFuncs(t *testing.T) {
 		})
 
 		Convey("With HTML", func() {
-			So(trimHTML(template.HTML("  A  B ")), ShouldEqual, "A  B")
+			So(trimHTML(template.HTML("  A  B ")), ShouldEqual, template.HTML("A  B"))
```

Comment:
I think maybe convert this test case to using html tag
`So(trimHTML(template.HTML("  <span>A  B</span> ")), ShouldEqual, template.HTML("<span>A  B</span>"))`